girder_worker/docker/transforms/__init__.py

```diff
@@ -318,6 +318,9 @@ def transform(self, *pargs, **kwargs):
         else:
             return os.path.join(self._volume.container_path, self.filename)
 
+    def _repr_model_(self):
+        return '<%s.%s: "%s">' % (self.__module__, self.__class__.__name__, self.filename)
```

Comment:
Seems odd that one `_repr_model_` uses the pattern `'str' % (args,)` while the other uses `u'str'.format()`. Note the different use of `u` prefix and `format`/`%`. Are we moving toward one method of formatting strings or are there reasons for the difference?

Reply:
I agree with Jeff that these should be consistent.  My preference would be for ```format``` but that is just because of personal habit,  not because of a reasoned argument.

Reply:
There are several ways to concatenate strings in python: `+`, `%`, `format`, `join`... Of these, `format` is the only one that does not automatically upgrade the output type to text type if one of the inputs happens to be text type. For that reason, we avoid using it in Girder and prefer `%` for formatting even though it lacks power -- if one forgets to prepend the `u` type specifier to the template string, it can cause exceptions to occur that won't be caught in testing unless you test with strings outside the range of ASCII.

This felt like a good time to break that rule because of the variable template. However, consistency is desirable. I'm fine with having them both use one way or the other, but if `format` is the way we want to go everywhere, we need to be disciplined about enforcing the `u` before the template string. I would go as far as wanting a lint rule for it if that's possible.

Reply:
An alternative would be to ```from __future__ import unicode_literals``` which would force all strings to be unicode literals unless they were specifically declared with b''. 

Currently the majority of the string formatting is done with ```%```  so we should probably stick with that for consistencies sake. 

As a side note - i'm of the opinion that automatically upgrading to text types is a _bug_ not a feature.  It is exactly the kind of behavior that allows Python 2 applications to be lazy about their bytes/text API, making converting to Python 3 difficult.

Reply:
> automatically upgrading to text types is a bug not a feature. It is exactly the kind of behavior that allows Python 2 applications to be lazy about their bytes/text API, making converting to Python 3 difficult.

I kind of agree with this, but not fully. I would fully agree if `unicode_literals` was the default behavior, but since it's not, I think this behavior creates too many pitfalls that won't be discovered until runtime.